pkg/asset/cluster/tfvars.go

```diff
@@ -433,6 +433,23 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 		if err != nil {
 			return err
 		}
+		con, err := ovirtconfig.NewConnection()
+		if err != nil {
+			return err
+		}
+		defer con.Close()
+
+		profiles, err := ovirtconfig.FetchVNICProfileByClusterNetwork(
+			installConfig.Config.Platform.Ovirt.ClusterID,
+			installConfig.Config.Platform.Ovirt.NetworkName,
+			con)
+		if err != nil {
+			return errors.Wrapf(err, "failed to compute values for oVirt platform")
+		}
+		if len(profiles) != 1 {
+			return errors.Wrapf(err, "failed to compute values for oVirt platform, there are multiple profiles.")
```

Comment:
If i understand correctly this is different default behavior than what you're replacing, but that behavior would be considered a bug so this is ok, in my opinion.

Reply:
right.